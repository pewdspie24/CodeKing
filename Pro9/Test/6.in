10
3010289728 6115095274
2468903087 4495278141
1951047301 2451486522
2900271624 4277652307
1470105321 3631302291
2252093208 2853659610
4218542893 6402835620
2112354593 3876666915
892118538 4315513208
3954992318 8242136522
