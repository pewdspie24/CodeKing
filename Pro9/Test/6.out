26854574
20639371
6790311
14436782
22832164
6090384
17648576
18517652
37143089
31598909
